docs/sources/panels/configure-thresholds/_index.md

```diff
@@ -0,0 +1,102 @@
+---
+title: 'Configure visualization thresholds'
+menuTitle: 'Configure visualization thresholds'
+description: 'This section includes information about using thresholds in your visualizations.'
+weight: 300
+aliases:
+  [
+    docs/grafana/latest/panels/thresholds/,
+    docs/grafana/latest/panels/,
+    docs/grafana/latest/panels/specify-thresholds/about-thresholds/,
+    docs/grafana/latest/panels/specify-thresholds/add-a-threshold/,
+    docs/grafana/latest/panels/specify-thresholds/add-threshold-to-graph/,
+    docs/grafana/latest/panels/specify-thresholds/delete-a-threshold/,
+  ]
+---
+
+# Configure visualization thresholds
+
+This section includes information about using thresholds in your visualizations. You'll learn about thresholds and their defaults, how to add or delete a threshold, and adding a threshold to a legacy panel.
+
+## About thresholds
+
+A threshold is a value that you specify for a metric that is visually reflected in a dashboard when the threshold value is met or exceeded.
+
+Thresholds provide one method for you to conditionally style and color your visualizations based on query results. You can apply thresholds to most, but not all, visualizations. For more information about visualizations, refer to [Visualization panels]({{< relref "../../visualizations" >}}).
+
+You can use thresholds to:
+
+- Color lines or areas in the [Graph panel]({{< relref "../../visualizations/graph-panel" >}})
+- Color lines in the [Time-series panel]({{< relref "../../visualizations/time-series/graph-color-scheme/#from-thresholds" >}})
+- Color the background or value text in the [Stat panel]({{< relref "../../visualizations/stat-panel" >}})
+- Color the gauge and threshold markers in the [Gauge panel]({{< relref "../../visualizations/gauge-panel" >}})
+- Color markers in the [Geomap panel]({{< relref "../../visualizations/geomap" >}})
+- Color cell text or background in the [Table panel]({{< relref "../../visualizations/table" >}})
+- Define regions and region colors in the [State timeline panel]({{< relref "../../visualizations/state-timeline" >}})
+
+There are two types of thresholds:
```

Comment:
The visualizations are in the visualization docs category (and not the Panels category), and we refer to them as visualizations in the UI, yet here we refer to visualizations with a "panel" suffix.

Reply:
corrected this, thanks